### DISCUSSION TITLE
Add new insulin profile Apidra (glulisine)

### DISCUSSION THREAD

**Discussion by kyletru** [NightscoutFoundation/xDrip#2932] [General]
Could we please add Apidra to the list of insulin profiles to add?

Thanks!

**Comment by Navid200** [comment]
Done

**Comment by Agio56** [comment]
@Navid200 

Has Apidra already been added? I can't find it in the settings.

**Reply by Navid200** [reply]
No, it has not been added.  I am sorry.
What I meant was that I have added the request to the list:
https://github.com/NightscoutFoundation/xDrip/issues/1898

**Reply by Agio56** [reply]
🙂👍
